web_src/less/_repository.less

```diff
@@ -622,6 +622,14 @@
       .issue-title {
         margin-bottom: .5rem;
 
+        code {
+          padding: 2px 4px;
+          font-size: .9em;
+          line-height: 1;
+          border-radius: 6px;
+          background-color: var(--color-neutral-muted);
+        }
```

Comment:
This block is copied from GitHub. So the question is:

* Why           `font-size: .9em; ` for Gitea?
* Could there already be an existing color instead of introducing new `--color-neutral-muted` from GitHub?

Reply:
Yes, we should use a existing color, there are plenty of options. Adding new colors should be avoided as it causes extra work for theme integrators.

Reply:
I suggest just going with no color definition at all, like normal markdown code blocks which just inherit.

Reply:
> This block is copied from GitHub. So the question is:
> 
> * Why           `font-size: .9em; ` for Gitea?
> * Could there already be an existing color instead of introducing new `--color-neutral-muted` from GitHub?

Removed font-size and used an existing color